Draft EIP: BLS12-381 Deterministic Account Hierarchy

CarlBeek opened the pull request:
This is the proposed standard for BLS12-381 Deterministic Account Hierarchy (Account Paths) for use within Eth2 as well as by the larger blockchain industry. It has existed in its own repo for a while now, but this is a better place to enshrine it as a standard for now.

axic commented on EIPS/eip-draft_bls_path.md:
```diff
@@ -0,0 +1,84 @@
+
+## Simple Summary
+
+This EIP defines the purpose of a given key, or family thereof, within a tree of keys. When combined with [eip-draft_bls_keygen.md](https://github.com/CarlBeek/EIPs/blob/bls_keygen/EIPS/eip-draft_bls_keygen.md), the combination of a seed and knowledge of the desired purpose of a key is sufficient to determine a key pair.
```
I believe these keygen links should point to #2333 ?

CarlBeek commented:
This is ready for merge after #2333 at which point CI should pass as the desired links will exist.

In the most recent commit I have squashed the commits and changed the `coin_type` [as a result of this suggestion](https://github.com/ethereum/EIPs/issues/2338#issuecomment-551446821) to 3600=60^2.

eip-automerger commented:
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):

 - Contains new file EIPS/eip-2334.md

Oleksandr-prog reviewed (commented):
l
